src/utils/rfqm_blockchain_utils.ts

```diff
@@ -0,0 +1,40 @@
+import { IZeroExContract } from '@0x/contract-wrappers';
+import { MetaTransaction, RfqOrder, Signature } from '@0x/protocol-utils';
+import { BigNumber } from '@0x/utils';
+
+import { NULL_ADDRESS } from '../constants';
+
+// allow a wide range for gas price for flexibility
+const MIN_GAS_PRICE = new BigNumber(0);
+// 10K Gwei
+const MAX_GAS_PRICE = new BigNumber(1e13);
+
+export class RfqBlockchainUtils {
+    constructor(private readonly _exchangeProxy: IZeroExContract) {}
+
+    public generateMetatransaction(
```

Comment:
perhaps add a comment that this method is used in the RFQM flow when 0x submits the transaction on chain (as opposed to the Maker submitting on chain)